care/facility/migrations/0485_migrate_prescriptions_and_medicineadministrations.py

```
@@ -0,0 +1,498 @@
+        return None
+    if prescription.frequency in FREQUENCY_MAP:
+        timing = FREQUENCY_MAP[prescription.frequency]
+        if prescription.days:
+            timing["repeat"]["bounds_duration"] = {
+                "value": prescription.days,
+                "unit": "d",
+            }
+        return timing
+
+
+def migrate_medication_request(MedicationRequest, prescription):
```

Ruff (0.8.2)

Too many branches (17 > 12)

(PLR0912)